doc/tutorials/imgproc/erosion_dilatation/erosion_dilatation.markdown

```diff
@@ -204,9 +204,9 @@ receives three arguments:
 
     We can choose any of three shapes for our kernel:
 
-    -   Rectangular box: CV_SHAPE_RECT
-    -   Cross: CV_SHAPE_CROSS
-    -   Ellipse: CV_SHAPE_ELLIPSE
+    -   Rectangular box: cv::SHAPE_RECT
```

Comment:
_(it is in Java tab)_
Rectangular box: Imgproc.SHAPE_RECT
Cross: Imgproc.SHAPE_CROSS
Ellipse: Imgproc.SHAPE_ELLIPSE